docs/source/index.rst

```diff
@@ -10,6 +10,10 @@ Installation
 
    $ pip install -U pydoctor
 
+Install with *restructuredtext* support::
+
+   $ pip install -U pydoctor docutils
+
```

Comment:
I'd prefer to implement #264, for a more reliable way of installing pydoctor with RST support.

Reply:
I agree, I also discovered that some directive need Sphinx actually not only docutils

Reply:
How does that work? Does Sphinx automatically register additional directive handlers for docutils, or should pydoctor make calls to Sphinx to set this up?

Reply:
It looks like it's the `.. code:: python` directive the issue:

```
docs/source/PyDoctor_ReST_Quick_Reference.py:169: bad docstring: Cannot analyze code. Pygments package not found.
```

Reply:
Hmm, we already have a Python syntax highlighter inherited from epydoc and I think Twisted contains another, so pulling in Pygments for that purpose would be a bit redundant. Would it be possible to register our own directive handler?

Another option would be to drop the epydoc highlighter and adopt Pygments ourselves.

Reply:
I see. The issue here is if we drop Pygments we need to say `.. code::` directive is not supported yet. And this PR is not about coding fancy directive for pydoctor yet neither (I haven't looked in details how to do so).
So I'll drop Pygments dep and say `.. code::` is not supported yet if that's ok with you